doc/help/elektra-spec.md

```diff
@@ -21,7 +21,7 @@ key database semantics (new plugins might be necessary, though).
 The most simple use is to enlist all keys that will be used by an application
 and maybe give a description for them (we use ini syntax in this document):
 
-```
+```ini
```

Comment:
> Does ronn, doxygen and the homepage render this correctly?

- `ronn`: Yes
- Doxygen: Yes
- Homepage: I hope so. The text looks correct in [Marked](http://marked2app.com) and the TextMate Markdown Preview (I use [Pandoc](http://pandoc.org) as Markdown processor).